apps/src/templates/teacherNavigation/LessonResources.tsx

```diff
@@ -28,13 +30,51 @@ const LessonResources: React.FC<LessonResourcesProps> = ({
   lessonNumber,
   lessonPlanUrl,
   lessonName,
+  standardsUrl,
+  vocabularyUrl,
 }) => {
   // Note that lessonPlanUrl is not needed for student resources
   // and should be null for student resoruces section
   const sectionHeaderText = lessonPlanUrl
     ? i18n.teacherResourcesforLessonMaterials()
     : i18n.studentResources();
 
+  const renderStandardsRow = () => {
+    if (!standardsUrl || !lessonPlanUrl) return null;
```

Comment:
Checking for this makes sure that if we are rendering the `StudentResources` this content is not included in the table.

Reply:
In my mind, this was an additional check for "hey, this should only be included in the `Teacher Resoruces` area not the `Student Resources` area.  It was more of a "tool to communicate to a reader" than a "needed for logic tool".  As I write that, I think it makes more sense to just explicitly communicate that in a comment.  I am modifying that now.